pdns/recursordist/rec-main.cc

```diff
@@ -885,6 +887,10 @@ static const char* toTimestampStringMilli(const struct timeval& tv, char* buf, s
 #ifdef HAVE_SYSTEMD
 static void loggerSDBackend(const Logging::Entry& entry)
 {
+  Logger::Urgency u = entry.d_priority ? Logger::Urgency(entry.d_priority) : Logger::Info;
+  if (u > s_logUrgency) {
+    return;
+  }
```

Comment:
Perhaps a comment of what's happening here could be helpful: "We don't need to log anything if the log level is at a higher urgency than the user-requested level".